source/upgrade.txt

```diff
@@ -0,0 +1,98 @@
+.. _c-upgrade:
+
+=======================
+Upgrade Driver Versions
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: compatibility, backwards compatibility
+
+Overview
+--------
+
+This page describes the changes you must make to your application
+when you upgrade to a new version of the {+driver-short+}.
+
+Before you upgrade, perform the following actions:
+
+.. TODO: Link to platform support page
+- Ensure the new {+driver-short+} version is compatible with the {+mdb-server+} versions
+  your application connects to and the platform your application runs on. For version 
+  compatibility information, see the :ref:`{+driver-short+} Compatibility <c-compatibility>` 
+  page.
+- Address any breaking changes between the current driver version
+  your application uses and your planned upgrade version in the
+  :ref:`Breaking Changes <c-breaking-changes>` section. To learn more about
+  {+mdb-server+} release compatibility changes, see the 
+  :ref:`Server Release Compatibility Changes <c-server-release-compatability-changes>` section.
+
+.. TODO: Add a tip to use the stable API and link to corresponding page.
```

Comment:
[note for reviewer] Wasn't sure where this page would live in the updated docs structure so left as a todo for now.